### DISCUSSION TITLE
I love this project

### DISCUSSION THREAD

**Discussion by TheBricktop** [pyunity/pyunity#65] [General]
Especially when the unity management are doing the usual pump and dump scheme with the whole company.  (google unity ironsource merge)

**Comment by rayzchen** [comment]
If you have any ideas on how to improve it, I'm all ears!

**Comment by TheBricktop** [comment]
Im curious about physics engine, how comparable is it to the physx in Unity?

**Comment by rayzchen** [comment]
***Well*** its very far off, though I am working on it, and I hope 0.9.0 (the next release) should finalize it. See #57 for more info.

**Reply by TheBricktop** [reply]
Did you considered using pyphysx or pybullet as a physics backend for this project? Rest will be writing a wrapper that is similar to unity way of handling  rigidbodies.

**Reply by rayzchen** [reply]
Never heard of that before, but I’ll look into it!

**Reply by rayzchen** [reply]
`pyphysx` has a huge dependency list, including matplotlib, scipy, numpy, numba and other things, which isnt necessarily lightweight... I did clone `hpp-fcl` and build it for Windows, which only requires numpy. Numpy is kinda big and slow to import.

`pybullet` is 80mb from source, which I need to install because they dont have wheels for Python 3.10 yet. It seems quite promising.
